releasedir/provider.go

```diff
@@ -103,6 +103,8 @@ func (p Provider) newBlobstore(dirPath string) boshblob.DigestBlobstore {
 		blobstore = boshblob.NewLocalBlobstore(p.fs, p.uuidGen, options)
 	case "s3":
 		blobstore = NewS3Blobstore(p.fs, p.uuidGen, options)
```

Comment:
Leaving unimplemented as discussed. There are SDK-level differences between GCS and S3 which make significantly more invasive than expected.